data/NV/incentives.json

```diff
@@ -1558,6 +1558,98 @@
     },
     "bonus_available": true
   },
+  {
+    "id": "NV-74",
+    "authority_type": "other",
+    "authority": "nv-csa-reno",
+    "eligible_geo_group": "nv-group-csa-reno",
+    "payment_methods": [
+      "pos_rebate"
```

Comment:
We categorize stuff like this as `assistance_program`, since it's more of a "you get free stuff" situation rather than a "you get all the money back" situation.